docs/checkit.js

```diff
@@ -130,6 +130,128 @@ Calendar.prototype.generateMonthObjects = function (startDate, endDate) {
 
 };
 
+// calendarAnalyzer takes a calendar object and can analyze facts about it.
+
+var CalendarAnalyzer = function (calendarState) {
+    "use strict";
```

Comment:
It's fine to do, I just thought you could get the whole file at once. Seems kind of annoying to have this scattered everywhere. JS is noisy enough.